platform/commonUI/browse/src/creation/CreateAction.js

```diff
@@ -88,22 +85,26 @@ define(
         CreateAction.prototype.perform = function () {
             var newModel = this.type.getInitialModel(),
                 parentObject = this.navigationService.getNavigation(),
-                newObject,
-                editableObject;
+                editorCapability,
+                newObject;
 
             newModel.type = this.type.getKey();
             newObject = parentObject.useCapability('instantiation', newModel);
-            editableObject = new EditableDomainObject(newObject, this.$q);
-            editableObject.setOriginalObject(parentObject);
-            editableObject.getCapability('status').set('editing', true);
-            editableObject.useCapability('mutation', function(model){
+            newObject.useCapability('mutation', function(model){
```

Comment:
We could simplify slightly by setting `newModel.location = parentObject.getId()` before the `instantiation`